File: RobustDemo/app/src/main/java/com/meta/robustdemo/SecondActivity.java
package com.meta.robustdemo;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.Button;

public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);

        Button btnSecondBug = findViewById(R.id.btn_second_bug);

        // btnSecondBug.setText(getSecondText());
        btnSecondBug.setText(getSecondText());
    }

    private String getSecondText() {
        return "second bug";
    }
}

File: GetLauncherPackageName/app/src/main/java/com/zhaopf/getlauncherpackagename/activitys/SecondActivity.kt
package com.zhaopf.getlauncherpackagename.activitys

import android.content.Intent
import android.os.Bundle
import com.zhaopf.getlauncherpackagename.R
import kotlinx.android.synthetic.main.activity_second.*

/**
 * 二级页面，用来判断是否是从桌面跳转过来的
 */
class SecondActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)

        btn_show_info.setOnClickListener {
            startActivity(Intent(this@SecondActivity, ThirdActivity::class.java))
        }
        // modify 222
    }
}